src/ProjectNeon/Assets/Features/Effects/AllEffects.cs

```diff
@@ -30,6 +30,10 @@ public static class AllEffects
         { EffectType.InterceptAttackForTurns, e => new ForNumberOfTurns(new InterceptAttack(), e.NumberOfTurns)},
         { EffectType.Attack, e => new Attack(e.FloatAmount)},
         { EffectType.EvadeAttacks, e => new Recurrent(new Evade(), e.IntAmount) },
+        { EffectType.Regenerate, e => new ForNumberOfTurns(new Heal(e.IntAmount), e.NumberOfTurns) },
```

Comment:
Does this trigger the effect at the start of each turn?

Should have a test to ensure that the character is correctly healed as turns advance.

Reply:
`ForNumberOfTurns` only keeps passive effects.

It doesn't apply Active effects.

No-one seems to be triggering this. 

This composition looks incomplete. If it is complete, then it can be proved with a test. But it looks dead wrong and and non-functional.